lib/manager.js

```diff
@@ -402,7 +402,6 @@ class Manager extends EventEmitter {
       : []
 
     if (!_isFunction(func)) {
-      debug('plugin %s is missing a handler for type %s', id, type)
```

Comment:
@tarcisiozf why we're removing this debug?

Reply:
this was a misconception when I first added this debug log, a plugin does not need to listen to all events in the manager lifecycle, this debug message floods the stdout with false positive errors.